StitchCore/StitchCore/Source/Core/StitchClient.swift

```diff
@@ -126,6 +129,11 @@ public class StitchClient: StitchClientType {
     public var isAuthenticated: Bool {
         return self.httpClient.isAuthenticated
     }
+    
+    // Returns the type of the provider used to log into the current session. nil if not authenticated
+    public var loggedInProviderType: String? {
+        return self.httpClient.loggedInProviderType
```

Comment:
It seems odd that `loggedInProviderType` is exposed via the HTTP client

Reply:
I did it this way since the HTTP Client is where Auth is actually cleared (and thus the lowest-level part of the client that touches the UserDefaults, but I guess  since nothing in the HTTP Client is accessing `loggedInProviderType` I can move it to `StitchClient`